### PR TITLE
[influxdb] Fix issues with the test project

### PR DESCRIPTION
**Pull request by CodeBlanch**
## Changes

* VS is complaining about the `OpenTelemetry.Exporter.InfluxDB.Tests` project having invalid targets.

### CODE REVIEW

**Review by utpilla — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Comment by Kielek** [comment]
@Havret, fyi. Merging.

File: test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/LineProtocolParser.cs [hunk 1]
```diff
@@ -94,6 +97,6 @@ private static DateTime ParseTimestamp(string? timestampSection)
         }
 
         long ticks = unixTimeNanoseconds / 100;
-        return DateTime.UnixEpoch.AddTicks(ticks);
```
Comment (Havret) [inline test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/LineProtocolParser.cs:97]: @CodeBlanch Why this change?

**Review by Havret — COMMENTED** [review]
lgtm

File: test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/LineProtocolParser.cs (hunk 1 above)
Comment (CodeBlanch) [reply test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/LineProtocolParser.cs:97]: `DateTime.UnixEpoch` is only available on [netstandard2.1 & netcoreapp2.1+](https://learn.microsoft.com/dotnet/api/system.datetime.unixepoch?view=net-7.0#applies-to) so I had to come up with some other solution for the .NET Framework targets on the project so it compiles. I decided just to use a local version of the value.
